dev-docs/AUTOMATE_DEV_ENV.md

```diff
@@ -0,0 +1,33 @@
+The goal of this document is to set up a development environment of 
+automate for Chef Infra Server related work. If you encounter a process
+or feature that isn’t straightforward or that took additional time for you to fully understand,
+consider writing an explanation here.
+
+## Setup
+
+To be able to build and run Automate follow the following steps
+1. Set up automate by following the docker setup in the [docs](https://github.com/chef/automate/blob/master/dev-docs/DEV_ENVIRONMENT.md#docker-setup)
+
+2. Configure Habitat
+
+   1. Habitat comes as part of Chef Workstation. Check if Habitat is already installed by running the command
+      `hab --version`. If not installed follow the [docs](https://docs.chef.io/habitat/install_habitat/)
+   2. Create an account in [Habitat](https://bldr.habitat.sh/#/sign-in) by signing in through Github.
+      Create an origin there and use the same as the default origin during setup.
+   3. To configure the Habitat cli run and follow the instructions: `hab cli setup`
+   4. Do not connect to the on-premise builder as you have already created one in [Habitat](https://bldr.habitat.sh).
+   6. Generate the [Personal Access token](https://bldr.habitat.sh/#/profile) and use the same during setup. 
+
+
+3. Finally, run `hab studio enter`.
+
+## Running Chef Infra Server in Automate dev env
+
+Automate is a unified chef product. It is technically a habitat project. There are many different components which make up the
+Automate project and each component is brough by by its associated `.hart` file. Amoung these components, few are of Chef Infra Server like bookshelf, nginx, bifrost and erchef. The following are the steps to build and deploy the components.
```

Comment:
It might be usefull to mention here that the automate repo contains wrappers for the code in the chef-server repo.